src/consumer/mod.rs

```diff
@@ -437,11 +437,11 @@ mod tests {
     };
     use log::LevelFilter;
     use regex::Regex;
-    #[cfg(feature = "tokio-runtime")]
+    #[cfg(any(feature = "tokio-runtime", feature = "tokio-rustls-runtime"))]
```

Comment:
Where do we define `tokio-rustls-runtime` feature?

Reply:
I saw the new definitions in `Cargo.toml` now.

Reply:
Yes, exactly :)